common.stone

```diff
@@ -28,21 +45,21 @@ union PathRoot
     member_home
         "Paths are relative to the authenticating team member's home
         directory. (This results in :field:`PathRootError.invalid' if the
-        user does not belong to a team.)"
+        user does not belong to a team.)."
```

Comment:
Nit: Remove final period (cf. http://www.thepunctuationguide.com/parentheses.html).

Reply:
I think this is a byproduct of some Linting jason added.